partitioner-config-service-api/src/main/proto/org/hypertrace/partitioner/config/service/v1/partitioner_config_service.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package org.hypertrace.partitioner.config.service.v1;
+
+option java_multiple_files = true;
+
+
+service PartitionerConfigService {
+
+  rpc GetPartitionerProfile(GetPartitionerProfileRequest) returns (GetPartitionerProfileResponse) {}
+
+  rpc PutPartitionerConfig(PutPartitionerConfigRequest) returns (PutPartitionerConfigResponse) {}
+
+  rpc GetAllPartitionerConfig(GetAllPartitionerConfigRequest) returns (GetAllPartitionerConfigResponse) {}
+
+  rpc DeletePartitionerConfig(DeletePartitionerConfigRequest) returns (DeletePartitionerConfigResponse) {}
+
+}
+
+message GetPartitionerProfileRequest {
+  string profile = 1; // not optional
+}
+
+message GetPartitionerProfileResponse {
+  PartitionerProfile profile = 1;
+}
+
+message PutPartitionerConfigRequest {
+  PartitionerProfile profile = 1;
+}
+
+message PutPartitionerConfigResponse {
+}
+
+message GetAllPartitionerConfigRequest {
+}
+
+message GetAllPartitionerConfigResponse {
+  repeated PartitionerProfile profiles = 1;
+}
+
+message DeletePartitionerConfigRequest {
+  string profile = 1;
+}
```

Comment:
Similarly, fix name and if you want, can make it plural.

```suggestion
message DeletePartitionerProfilesRequest {
  repeated string profile_names = 1;
}
```

Reply:
done